pygments/lexers/configs.py

```diff
@@ -968,9 +968,11 @@ class TOMLLexer(RegexLexer):
 
     tokens = {
         'root': [
+            # Table
+            (r'^(\s*)(\[.*?\])$', bygroups(Text, Keyword)),
 
             # Basics, comments, strings
-            (r'\s+', Text),
+            (r'[ \t]+', Text),
```

Comment:
is this change necessary? You're not matching newlines between entries anywhere now; they are getting caught by the "out of rules" handler in lexer.py.

Reply:
Yes, otherwise tables with preceding spaces are not found. I guess `\s+` consumes the newline character.

<img width="376" alt="image" src="https://user-images.githubusercontent.com/176810/127441648-24f57633-c11e-4927-ab3a-65c97f6c8b7d.png">

<img width="376" alt="image" src="https://user-images.githubusercontent.com/176810/127441706-a950fec7-374a-401b-9e56-4b86ca2aff5c.png">

Reply:
Right, but then please include an explicit rule for `'\n'` too.

Reply:
Done.

Reply:
Anything else needed?